core/server/models/user.js

```diff
@@ -110,6 +120,29 @@ User = ghostBookshelf.Model.extend({
             })();
         }
 
+        /**
+         * CASE: add model, hash password
+         * CASE: update model, hash password
+         *
+         * Important:
+         *   - Password hashing happens when we import a database
```

Comment:
This behaviour is not a new. When importing a User with a password, the password get's hashed again.
I wanted to ask why we are doing that?